src/main/kotlin/calculator/domain/Expression.kt

```diff
@@ -0,0 +1,51 @@
+package calculator.domain
+
+class Expression(input: String) {
+    var values = listOf<String>()
```

Comment:
init block을 활용하셔도 좋고 가변 인자(var)를 사용하셔도 좋지만

set도 private으로 막혀있고 init 이외에 values를 변경하는 부분은 없는 것 같습니다. 여기서는 가변 인자일 필요가 없다고 생각이 드네요. 

그리고 init block 내에 로직에 대해서 함수로 추출하면 함수의 이름을 통해 어떤 역할을 수행하는지 유추할 수 있습니다. init block보다는 함수로 추출하고 아래와 같이 사용해보셔도 좋을 것 같아요.

```suggestion
    val values = generateValues()
```